content/blog/2019-11-15-the-state-of-r-in-africa.md

```diff
@@ -0,0 +1,33 @@
+---
+title: The-state-of-R-in-Africa
+banner:"/images/blog/The-state-of-R-in-Africa/banner.jpg"
+author: Lorna Maria
+date: '2019-11-15'
+description: "2019 Round-up of initiatives in the R community across Africa."
+slug: the-state-of-r-in-africa
+categories: ["news"]
+tags: ["community"]
+tocify: no
+---
+##Growth of the R ecosystem in Africa 
+
+Over the past 2 years, we have seen R transform from just an academic and health care research language to one of the major languages in the budding data science community in Africa. For a long time, data scientists and related disciplines have relied on R and many other packages to analyse data, however, the broader developer community didn’t fully understand its power in the tech industry.
+An effective model to enter the Africa developer ecosystem relies on a small group of enthusiasts creating a safe learning environment, documenting and sharing their learning experiences and slowly attracting interest learners. Africa R users group are using this model to introduce R as a language for data science by organising communities across different cities in Africa.
+
+##Evolution of chapters and representation
+The growth of technical talent in Africa positively correlates with the growth of communities around certain technologies/tools/languages. These communities are creating a long table to share opportunities on the continent. When we shift the spotlight to R in the Africa tech community we see an evolution useRs groups of not only academic researchers completing their PhD or master’s degrees but also people using R at their workplaces or for their freelance projects. 
+There is a unique representation in the Africa R community that shows how the useRs have created a safe space for equal active participation. Global communities like R Ladies and Women in Machine Learning and Data Science have given so many women in the community a platform to start R user groups in their localities. The support from these groups has contributed to the early success of R user groups as the safest places to start your data science career.
+![R Ladies at the Annual Women Techmakers Event](/images/blog/The-state-of-R-in-Africa/representation.jpg)
+
+Some of the chapters include [Kampala](https://mobile.twitter.com/Rladieskampala),[Lagos](https://mobile.twitter.com/LagosRUsers),[Dar es salaam](https://mobile.twitter.com/daR_users), [Abidjan](https://mobile.twitter.com/AbidjanRusers), [Johannesburg](https://mobile.twitter.com/RLadiesJozi),[DRC] (https://mobile.twitter.com/DrcUsers),[Cotonou](https://mobile.twitter.com/RLadiesCtn) among others. 
```

Comment:
Perhaps make clear which are general R User Groups and which are R-Ladies groups?